apollo-portal/templates/deployment-portal.yaml

```diff
@@ -60,6 +60,8 @@ spec:
     metadata:
       labels:
         app: {{ $portalFullName }}
+      annotations:
```

Comment:
I think it's better to not add this section if annotations is empty?

Reply:
I agree it.
I had changed it with block